static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -112,17 +112,18 @@ export function TimeSeriesWidgetVisualization(props: TimeSeriesWidgetVisualizati
     releaseBubbleXAxis,
     releaseBubbleGrid,
   } = useReleaseBubbles({
-    chartRef,
     chartRenderer: ({start: trimStart, end: trimEnd}) => {
       return (
-        <TimeSeriesWidgetVisualization
-          {...props}
-          disableReleaseNavigation
-          plottables={props.plottables.map(plottable =>
-            plottable.constrain(trimStart, trimEnd)
-          )}
-          showReleaseAs="line"
-        />
+        <div style={{height: '220px'}}>
```

Comment:
I had to remove [the height from the drawer](https://github.com/getsentry/sentry/pull/86950/files#diff-b4cafa3a7a07668ef808d5712280fc4980f176e360d11a45a3a522933d9b81d2L87) to support the issue details chart which uses the default 100px height.